.github/ISSUE_TEMPLATE/new_webpage.yml

```
@@ -0,0 +1,112 @@
+name: ✨ New Webpage
+description: Start here to request a new page
+title: "✨ {{ TITLE }} "
```

```suggestion
title: "📄 {{ TITLE }} "
```
Should we use the page emoji instead? 📄

Sparkle is meant to be used for features (code)